hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -283,7 +275,7 @@ public String toString() {
 		}
 		sb.append("\n  Congestion level start times are           ::");
 		for (var start : congestionLevelStarts) {
-			sb.append("\n    ").append(reprOf(start));
+			sb.append("\n    ").append(start);
```

Comment:
Same as line 259 (originally line 267)!

Reply:
Done.

Reply:
This is still not fixed yet!

Reply:
Fixed now. Was looking at the wrong branch.